front/app/containers/Admin/projects/all/index.tsx

```diff
@@ -241,6 +272,27 @@ const AdminProjectsList = memo(({ className }: Props) => {
               active={activeTab === 'archived'}
               url="/admin/projects/archived"
             />
+            {isProjectReviewEnabled && isAdmin(authUser) && (
```

Comment:
So we don't show those tabs to folders moderators?

Reply:
Not for now as it makes it complicated to show only relevant projects in those tabs. 
We might reconsider moving forward, though.

Reply:
Maybe something we should document in the FAQs?

Reply:
Good idea! I'll add it :)